iocore/net/YamlSNIConfig.cc

```diff
@@ -213,6 +213,38 @@ std::set<std::string> valid_sni_config_keys = {TS_fqdn,
 namespace YAML
 {
 template <> struct convert<YamlSNIConfig::Item> {
+  static ts::port_range_t
+  parse_single_inbound_port_range(Node const &node, swoc::TextView port_view)
+  {
+    auto min{port_view.split_prefix_at('-')};
+    if (!min) {
+      min = port_view;
+    }
+    auto const &max{port_view};
+
+    swoc::TextView parsed_min;
+    long min_port{swoc::svtoi(min, &parsed_min)};
+    swoc::TextView parsed_max;
+    long max_port{swoc::svtoi(max, &parsed_max)};
```

Comment:
Use `auto max_port = ...`

Reply:
What's the reason for using copy initialization over universal initialization in this case?

Reply:
There are plenty of style guides that recommend `=` over uniform initialization for trivial initialization like this, especially with `auto` where narrowing conversions are not a concern.  Style-wise, in ATS, this is certainly the norm.